src/main/java/net/sf/jabref/gui/util/comparator/IconComparator.java

```diff
@@ -17,13 +17,14 @@
 
 import net.sf.jabref.model.entry.BibEntry;
 
+import java.io.Serializable;
 import java.util.Comparator;
 import java.util.List;
 
 /**
  * Comparator that handles icon columns.
  */
-public class IconComparator implements Comparator<BibEntry> {
+public class IconComparator implements Comparator<BibEntry>, Serializable {
```

Comment:
I meant not to change this. Seems like the files were not correctly overwritten when I deleted the old branch from the previous PR.